### ISSUE TITLE
[BUG] `import/no-unused-modules` does not work with `import { name as otherName }`

### ISSUE THREAD

**Issue by ehmicky** [benmosher/eslint-plugin-import#1339]
When using `import { ... as ... }` statements, `import/no-unused-modules` does not seem to properly work.

In the following example, `test` export is used, but it's still reported as not used:

```
$ eslint .
/home/ehmicky/no-unused-modules-bug/src/file.js
  1:1  error  exported declaration 'test' not used within other modules  import/no-unused-modules

✖ 1 problem (1 error, 0 warnings)
```

`src/main.js`:

```js
import { test as testTwo } from './file.js'
```

`src/file.js`:

```js
export const test = true
```

`.eslintrc.json`:

```json
{
  "parserOptions": {
    "ecmaVersion": 2019,
    "sourceType": "module"
  },
  "plugins": ["import"],
  "rules": {
    "import/no-unused-modules": [
      2,
      {
        "unusedExports": true,
        "ignoreExports": ["src/main.js"]
      }
    ]
  }
}
```

`package.json`:

```json
{
  "name": "no-unused-modules-bug",
  "version": "1.0.0",
  "dependencies": {},
  "devDependencies": {
    "eslint": "^5.16.0",
    "eslint-plugin-import": "^2.17.2"
  }
}
```

Note: removing the `ignoreExports: ["src/main.js"]` does not fix this issue.

**Comment by ljharb** [comment]
cc @rfermann

**Comment by ehmicky** [comment]
Thanks a lot for the quick response!

However it seems like this bug is still happening. If you publish a new version of this library, I can create a demo repository.

**Comment by rfermann** [comment]
@ehmicky: a demo repo would be great.

**Comment by ehmicky** [comment]
Sure I can do that.

The repo will use the latest published version though, so won't include your first fix. But if you manually edit the dependency, you should see that this fix does not modify the behavior unfortunately.

**Comment by ehmicky** [comment]
Demo repo: https://github.com/ehmicky/no-unused-modules-bug

**Comment by rfermann** [comment]
Thanks for the repo. As it turned out, I missed one piece of code :sweat_smile:

**Comment by ehmicky** [comment]
It works, thanks @rfermann! :tada:
